tests/test_label_change.py

```diff
@@ -0,0 +1,7 @@
+import os
+
+
+def test_label_change():
```

Comment:
Related to the comment above, I think we should just be testing the args parser.